packages/blue-sdk/src/market/Market.ts

```diff
@@ -18,6 +18,11 @@ export interface CapacityLimit {
   limiter: CapacityLimitReason;
 }
 
+export interface MaxCapacitiesOptions {
+  borrow?: { maxLtv?: bigint };
+  withdrawCollateral?: { maxLtv?: bigint };
+}
```

Comment:
If defined, this should be defined at the `MarketUtils` level imo (because that's where the function is actually implemented) and in this case it is more semantically equal to `Partial<Pick<MarketConfig, "lltv">>`